watchdog/src/lib.rs

```diff
@@ -35,6 +38,11 @@ thread_local! {
 /// This function is called when the canister is created.
 #[init]
 fn init() {
+    // Setup the stdlib hooks.
+    #[cfg(target_arch = "wasm32")]
+    printer::hook();
```

Comment:
If we do this, doesn't it mean we can just use the version from cdk? Why do we still need to copy the code into this printer module?

Reply:
The hook from cdk is not a public method, it's used inside `setup` method behind the flag. 
I don't fully understand the effect of such a flag, especially because in bitcoin canister the hook has to be set up both in init and in post_upgrade methods. To make sure it's set up both times I suppose it's better to have our own copy without a flag.